### PR TITLE
CSE release 1.64 control-plane-manager src-artifact

### PR DESCRIPTION
**Pull request by mvasl**

### CODE REVIEW

File: modules/040-control-plane-manager/images/control-plane-manager/werf.inc.yaml [hunk 1]
```diff
@@ -21,15 +21,23 @@ docker:
   ENTRYPOINT: ["/controller"]
 {{- end }}
 ---
-artifact: {{ $.ModuleName }}/controller-artifact
-from: {{ $.Images.BASE_GOLANG_22_ALPINE_DEV }}
-fromCacheVersion: 1
+artifact: {{ $.ModuleName }}-{{ $.ImageName }}-src-artifact
```
Comment (RomanenkoDenys) [inline modules/040-control-plane-manager/images/control-plane-manager/werf.inc.yaml:24]: we use another naming -  {{ $.ModuleName }}/{{ $.ImageName }}-src-artifact

File: modules/040-control-plane-manager/images/control-plane-manager/werf.inc.yaml [hunk 2]
```diff
@@ -21,15 +21,23 @@ docker:
...
 git:
   - add: /{{ $.ModulePath }}modules/040-{{ $.ModuleName }}/images/{{ $.ImageName }}/controller
-    to: /
+    to: /src
     stageDependencies:
       install:
         - '**/*'
+---
+artifact: {{ $.ModuleName }}/controller-artifact
+from: {{ $.Images.BASE_GOLANG_22_ALPINE_DEV }}
```
Comment (RomanenkoDenys) [inline modules/040-control-plane-manager/images/control-plane-manager/werf.inc.yaml:35]: Please use BASE_GOLANG_23_ALPINE with proxy

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
some nits

File: modules/040-control-plane-manager/images/control-plane-manager/werf.inc.yaml (hunk 2 above)
Comment (mvasl) [reply modules/040-control-plane-manager/images/control-plane-manager/werf.inc.yaml]: done

File: modules/040-control-plane-manager/images/control-plane-manager/werf.inc.yaml (hunk 1 above)
Comment (mvasl) [reply modules/040-control-plane-manager/images/control-plane-manager/werf.inc.yaml]: fixed

File: modules/040-control-plane-manager/images/etcd/werf.inc.yaml [hunk 3]
```diff
@@ -1,30 +1,39 @@
 ---
+artifact: {{ $.ModuleName }}/{{ $.ImageName }}-src-artifact
+fromArtifact: src-artifact
+shell:
+  beforeInstall:
+  - git clone -b v3.5.9 --depth 1 {{ $.SOURCE_REPO }}/etcd-io/etcd.git /src
```
Comment (RomanenkoDenys) [inline modules/040-control-plane-manager/images/etcd/werf.inc.yaml:6]:
```suggestion
  - git clone -b v3.5.16 --depth 1 {{ $.SOURCE_REPO }}/etcd-io/etcd.git /src
  - rm -rf /src/tools
```

**Pull request merged**